engine/table/src/main/java/io/deephaven/engine/table/impl/util/compact/CompactKernelImpl.java

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl.util.compact;
+
+import io.deephaven.util.QueryConstants;
+
+final class CompactKernelImpl {
+    public static boolean isNullOrNan(byte x) {
```

Comment:
I feel like using replication was the better answer here. I don't mind renaming to `isNullOrNan` for precision.